hawk/app/models/report.rb

```diff
@@ -1,69 +1,226 @@
 # Copyright (c) 2009-2015 Tim Serong <tserong@suse.com>
 # See COPYING for license.
 
+require 'fileutils'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.